scripts/secrets_manager

```diff
@@ -6,34 +6,58 @@ function check_dependencies() {
   echo '----------------------------'
 
   if ! [ -x "$(command -v age)" ]; then
-  	echo 'age is not installed. installing with `brew install age`' >&2
-  	brew install age;
+    echo 'age is not installed. installing with `brew install age`' >&2
+    brew install age;
 
     if ! [ -x "$(command -v age)" ]; then
-  	  echo 'age is still not installed. installing with `brew install age`' >&2
-  	  exit 1
-	  else 
-      echo 'all dependencies installed' 
+      echo 'age is still not installed. installing with `brew install age`' >&2
+      exit 1
     fi
-	else 
-    echo 'all dependencies installed' 
-	fi
+  fi
+
+  if ! [ -x "$(command -v rename)" ]; then
+    echo 'rename is not installed. installing with `brew install rename`' >&2
+    brew install rename;
 
+    if ! [ -x "$(command -v rename)" ]; then
+      echo 'rename is still not installed. installing with `brew install rename`' >&2
+      exit 1
+    fi
+  fi
+
+  echo "All dependencies installed"
 }
 
 function check_dependencies_silent() {
   # Only emits messages upon failure & exit
```

Comment:
@the-real-adammork We should DRY this out and have one function that can be configured to log or not log output via a param